docs/resources/ui-building-blocks/pages/page-elements.md

```diff
@@ -0,0 +1,414 @@
+---
+title: Page/Scaffold Elements
+slug: scaffold-elements
+sidebar_position: 2
+toc_max_heading_level: 5
+---
+
+# Scaffold Elements
+
+:::info
+To learn about Scaffold and the various page elements under it, refer to the [Behind a
+FlutterFlow Page](pages.md#behind-a-flutterflow-page) section.
```

Comment:
1. The link path is not found
2. [**Behind a FlutterFlow Page**] bold

Reply:
changed to bold, but page isnt written yet